__tests__/lib/hast.test.ts

```diff
@@ -19,7 +20,8 @@ describe('hast transformer', () => {
       h('h2', { id: 'its-coming-from-within-the-component' }, "It's coming from within the component!"),
     );
 
-    // @ts-ignore
+    // @ts-expect-error - the custom matcher types are not being set up
+    // correctly
```

Comment:
This appears to be happening because the `expect` here is coming from `@types/jest` and the custom matcher is being set up on the `expect` that Vitest offers. We can clean this up when we excise Jest from this after moving puppeteer tests over to Vitest browser testing.